### PR TITLE
Synchronize with BCD v5.6.18

### PR DESCRIPTION
**Pull request by OnkarRuikar**
[BCD release v5.6.16](https://github.com/mdn/browser-compat-data/releases/tag/v5.6.16)
[BCD release v5.6.17](https://github.com/mdn/browser-compat-data/releases/tag/v5.6.17)
[BCD release v5.6.18](https://github.com/mdn/browser-compat-data/releases/tag/v5.6.18)

### CODE REVIEW

**Review by chrisdavidmills — COMMENTED** [review]
Thanks for all this work, @OnkarRuikar.

The content updates all look sound, but I did have a question — all the WebGPU stuff has been supported in Chromium for a while, but it is now supported in Gecko/WebKit in preview/nightly. Doesn't this still mean it counts as experimental, or no?

Chrome 113 was released in April 2023, so it hasn't been supported there for over two years (I'm looking at our definition of experimental: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete)

File: files/en-us/web/api/delegatedinktrailpresenter/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The **`DelegatedInkTrailPresenter`** interface of the [Ink API](/en-US/docs/Web/
 
 ## Instance properties
 
-- {{domxref("DelegatedInkTrailPresenter.expectedImprovement", "expectedImprovement")}} {{Deprecated_Inline}} {{Non-standard_Inline}} {{ReadOnlyInline}}
+- {{domxref("DelegatedInkTrailPresenter.expectedImprovement", "expectedImprovement")}} {{Deprecated_Inline}} {{Non-standard_Inline}} {{ReadOnlyInline}} {{experimental_inline}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/delegatedinktrailpresenter/index.md:18]: https://github.com/mdn/browser-compat-data/blob/main/api/DelegatedInkTrailPresenter.json#L39-L76 is mark as not experimental

**Comment by OnkarRuikar** [comment]
> all the WebGPU stuff has been supported in Chromium for a while, but it is now supported in Gecko/WebKit in preview/nightly. Doesn't this still mean it counts as experimental, or no?

@chrisdavidmills You'll have to take this up with BCD folks. The conversation is happening in the corresponding PR in browser-compat-data repo: https://github.com/mdn/browser-compat-data/pull/25047#issuecomment-2483500175 . Looks like you were CCed on the BCD side PR.

File: files/en-us/web/api/delegatedinktrailpresenter/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/delegatedinktrailpresenter/index.md:18]:
```suggestion
- {{domxref("DelegatedInkTrailPresenter.expectedImprovement", "expectedImprovement")}} {{Deprecated_Inline}} {{Non-standard_Inline}} {{ReadOnlyInline}}
```

**Comment by OnkarRuikar** [comment]
- @chrisdavidmills The BCD PR https://github.com/mdn/browser-compat-data/pull/25135 fixed the WebGPU issues. Now, this sync has become a lot smaller.

Regarding issue pointed by Sky, the bug has been fixed in the sync bot. Remaining occurrences in content have been fixed in the [previous commit](https://github.com/mdn/content/pull/36856/commits/1d53a29997077425ffdbc9b9c4004dc82d498c0c).

**Review by bsmth — APPROVED** [review]
Thanks, all! Leaving a +1 on this 👍🏻

**Pull request merged**